src/PointwiseFunctions/AnalyticSolutions/RelativisticEuler/Tov.cpp

```diff
@@ -220,17 +216,15 @@ void TovSolution::integrate(
       // Transform observed radius to isotropic, so we use the isotropic radius
       // for all interpolations below
       observer.radius[i] /= square(observer.conformal_factor[i]);
-      // The interpolation is not safe otherwise
     }
+    // The interpolation is not safe otherwise
     observer.radius.back() = outer_radius_;
     conformal_factor_interpolant_ = intrp::BarycentricRational(
         observer.radius, observer.conformal_factor, 5);
   }
 
   mass_over_radius_interpolant_ =
       intrp::CubicSpline(observer.radius, observer.mass_over_radius);
```

Comment:
@isaaclegred why was this changed to cubic spline? The answer is probably in #4111 but that's a very long thread.

Reply:
I think it was 1) stability and 2) smoothnees of the interpolation. I think either the r=0 or outer radius was going unstable.

Reply:
Yes, it had to do with stability when many steps are required to reach the boundary of the star.  I didn't know enough about barycentric rational interpolation to figure out exactly what was up, but after something like a radius of 100000000 M_sun came out of the TOV solver I concluded it was probably a pole in the interpolant.  This was a problem primarily for the phase-transition star, where many many steps are needed to get high precision.  You might be able to fix this by downsampling the dense output further?  I'm not sure.